service/src/main/resources/db/changesets/20221012_initial.yaml

```diff
@@ -23,4 +23,4 @@ databaseChangeLog:
         -  sqlFile:
             splitStatements:  false
             relativeToChangelogFile: true
-            path:  functions/pathfunc.sql
+            path: functions/pathfunc.sql
```

Comment:
I would revert this. In case liquibase hashes this file, or something.

Reply:
Thanks